### PR TITLE
#4 Add getReport method

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/epir-core/issues/4 |
| Dependencies | https://github.com/hivesolutions/yonius/pull/18 |
| Decisions | • Add `getReport` method instead of mock method. <br> • Add hermes mock order <br> • Adapt example to call `getReport`. <br> • Add issue and bug report template.  |

### CODE REVIEW

File: examples/simple.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 const epirPapyrus = require("..");
+const mockOrder = require("../mocks/hermes/bastia.json");
+
+const baseUrl = "http://localhost:3000/";
```
Comment (gcandal) [inline examples/simple.js:4]: Point to now instead

File: js/report.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const ReportAPI = superclass =>
+    class extends superclass {
+        async getReport(name, order) {
+            const url = this.baseUrl + `reports/${name}`;
+            const options = { body: order };
+            const contents = await this.post(url, options);
+            const reportHtml = await contents.text();
```
Comment (gcandal) [inline js/report.js:7]: Delete

File: mocks/hermes/bastia.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
+{
```
Comment (gcandal) [inline mocks/hermes/bastia.json:1]: Move this inside `examples`

**Review by gcandal — CHANGES_REQUESTED** [review]

**Review by gcandal — APPROVED** [review]

File: examples/simple.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
 const epirPapyrus = require("..");
+const mockOrder = require("./mocks/hermes/bastia.json");
+
+const baseUrl = "https://epir-papyrus-now.platforme.com/";
```
Comment (joamag) [inline examples/simple.js]: remove base url

**Review by joamag — APPROVED** [review]

**Pull request merged**